Enterprise-CMCS/macpro-mdct-mcr
# refactor NumberField to use CmsdsTextField

### Description
refactor NumberField to use CmsdsTextField instead of the custom TextField component. Also refactors some styling logic.


### Related ticket(s)


MDCT-1918

---
### How to test
Go to any number field and make sure it still functions as expected. Sanctions have a percentage field to check on the styling refactor.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
